### ISSUE TITLE
NTR:  intranuclear rod assembly

### ISSUE THREAD

**Issue by rjdodson** [geneontology/go-ontology#12950]
Hello:

This term is related to the term I requested last week "intranuclear rod". I unsuccessfully tried to create it with TermGenie, so am submitting as an NTR. Let me know of any questions. 

Thanks,
Bob

Name: intranuclear rod assembly
Ontology: process
Synonyms: intranuclear actin rod assembly
Definition: The assembly of an intranuclear rod; intranuclear rods are macromolecular fibers consisting of actin and cofilin that is formed in the nucleus as a consequence of chemical or mechanical stress conditions.

Child of: GO:0022607  cellular component assembly

References:
PMID:28074884 Actin-Interacting Protein 1 Contributes to Intranuclear Rod Assembly in Dictyostelium discoideum

**Comment by ukemi** [comment]
Hi Rob,

I successfully created this with Term Genie. Were you getting an error message?

-D

**Comment by rjdodson** [comment]
Hi David:

Yes, I got an error message, but have closed the window and can't remember the exact error message. I used the cc_assembly_disassembly template and the error had something to do with not finding the component "intranuclear rod" which I had typed in. I could have spelled it wrong, or had an extra space...who knows. 

Anyway, thanks for creating the new GO term(s). I will add these as soon as they are available in protein2GO.

Bob
